Content.Server/Construction/Components/WelderRefinableComponent.cs

```diff
@@ -11,12 +12,12 @@ namespace Content.Server.Construction.Components
     public sealed partial class WelderRefinableComponent : Component
     {
         [DataField("refineResult")]
```

Comment:
```suggestion
        [DataField]
```

Reply:
DataFields no longer need explicit strings to go along with them, so you should probably remove them whenever you modify a system. 

Then again that might just be EE brain, I dunno if this codebase has explicit rules on modifying upstream files or anything.